multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -140,6 +173,18 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		// Nothing to handle in this case and MemberClusterAnnounce will also be deleted soon.
 		// TODO: Add ClusterClaim webhook to make sure it cannot be deleted while ClusterSet is present.
 	}
+
+	_, installed, _ := r.installedMemberClusterAnnounce.GetByKey(req.NamespacedName.String())
+	if !installed {
```

Comment:
So we do not support update still? You will add a webhook for that?

@luolanzone : today in any case (e.g. ClusterSet or ClusterClaim changes) member cluster will update MemberAnnounce?

Reply:
I double checked the code, If ClusterID of leader in ClusterSet is changed, it will rebuilt a new remote common area and update the leader cluster ID in memberClusterAnnounce. I will check and add ClusterClaim and ClusterSet webhook to make sure no update is allowed.